python/gtsam/examples/TranslationAveragingExample.py

```diff
@@ -0,0 +1,122 @@
+"""
+GTSAM Copyright 2010-2018, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+See LICENSE for the license information
+
+This example shows how 1dsfm uses outlier rejection (MFAS) and optimization (translation recovery)
+together for estimating global translations from relative translation directions and global rotations.
+The purpose of this example is to illustrate the connection between these two classes using a small SfM dataset.
+
+Author: Akshay Krishnan
+Date: September 2020
+"""
+
+from collections import defaultdict
+from typing import Tuple, List
+
+import numpy as np
+
+import gtsam
+from gtsam.examples import SFMdata
+
+
+def get_data() -> Tuple[gtsam.Values, List[gtsam.BinaryMeasurementUnit3]]:
+    """"Returns data from SfMData.createPoses(). This contains global rotations and unit translations directions."""
```

Comment:
i might add a little more here, or in the module docstring, explaining where the data comes from. Are we perturbing known poses with noise, and then trying to recover the ground truth from the noisy rotations?

Reply:
I think we can still improve this docstring -- i don't think it explains all that is happening here in this function